src/controller/proxy/local_test.go

```diff
@@ -85,7 +85,7 @@ func (lh *localHelperTestSuite) TestBlobExist_False() {
 	lh.registryClient.On("BlobExist", mock.Anything, mock.Anything).Return(false, nil)
 	exist, err := lh.local.BlobExist(ctx, art)
 	lh.Require().Nil(err)
-	lh.Assert().Equal(false, exist)
```

Comment:
I like it, It spotted already the typical error with assertions and boolean logic.

```
	lh.Assert().Equal(false, exist)
	lh.Assert().False(exist)
```

@mmorel-35 is the only thing it finds